timeseries/src/autogluon/timeseries/models/chronos/pipeline/chronos.py

```diff
@@ -11,14 +11,17 @@
 
 import torch
 import torch.nn as nn
-from transformers import AutoConfig, AutoModelForCausalLM, AutoModelForSeq2SeqLM, GenerationConfig, PreTrainedModel
+from transformers import AutoConfig, AutoModelForSeq2SeqLM, GenerationConfig, PreTrainedModel
 
 from autogluon.timeseries.utils.warning_filters import set_loggers_level
 
+from . import ForecastPipeline, ForecastType
```

Comment:
```suggestion
from .forecast_pipeline import ForecastPipeline, ForecastType
```
Does the current code result in circular imports error?